code/game/gamemodes/wizard/artefact.dm

```diff
@@ -597,3 +597,231 @@ var/global/list/multiverse = list()
 	probability_evil = 100
 	duplicate_self = 1
 	sword_type = /obj/item/weapon/multisword/pike
+
+
+/////////////////////////////////////////Necromantic Stone///////////////////
+
+/obj/item/device/necromantic_stone
+	name = "necromantic stone"
+	desc = "A shard capable of resurrecting humans as skeleton thralls."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "necrostone"
+	item_state = "electronic"
+	origin_tech = "bluespace=4;materials=4"
+	w_class = 1
+	var/list/spooky_scaries = list()
+	var/unlimited = 0
+	var/heresy = 0
+
+/obj/item/device/necromantic_stone/unlimited
+	unlimited = 1
+
+/obj/item/device/necromantic_stone/attack(mob/living/carbon/human/M as mob, mob/living/carbon/human/user as mob)
+
+	if(!istype(M))
+		return ..()
+
+	if(!istype(user))
+		return
+
+	if(M.stat != DEAD)
+		user << "<span class='warning'>This artifact can only affect the dead!</span>"
+		return
+
+	if(!M.mind || !M.client)
+		user << "<span class='warning'>There is no soul connected to this body...</span>"
+		return
+
+	check_spooky()//clean out/refresh the list
+
+	if(spooky_scaries.len >= 3 && !unlimited)
+		user << "<span class='warning'>This artifact can only affect three undead at a time!</span>"
+		return
+	M.makeSkeleton()
+	M.visible_message("<span class = 'warning'> A massive amount of flesh sloughs off [M] and a skeleton rises up!</span>")
+	M.revive()
+	spooky_scaries |= M
+	M << "<span class='userdanger'>You have been revived by </span><B>[user.real_name]!</B>"
+	M << "<span class='userdanger'>They are your master now, assist them even if it costs you your new life!</span>"
+	equip_skeleton(M)
+	desc = "A shard capable of resurrecting humans as skeleton thralls[unlimited ? "." : ", [spooky_scaries.len]/3 active thralls."]"
+
+/obj/item/device/necromantic_stone/proc/check_spooky()
+	if(unlimited) //no point, the list isn't used.
+		return
+	for(var/X in spooky_scaries)
+		if(!istype(X, /mob/living/carbon/human))
+			spooky_scaries.Remove(X)
+			continue
+		var/mob/living/carbon/human/H = X
+		if(H.stat == DEAD)
+			spooky_scaries.Remove(X)
+			continue
+	listclearnulls(spooky_scaries)
+
+//Funny gimmick, skeletons always seem to wear roman/ancient armour
+//Voodoo Zombie Pirates added for paradise
+/obj/item/device/necromantic_stone/proc/equip_skeleton(mob/living/carbon/human/H as mob)
+	for(var/obj/item/I in H)
+		H.unEquip(I)
+	var/randomSpooky = "roman"//defualt
+	if(heresy)
+		randomSpooky = "yand"
+	else
+		randomSpooky = pick("roman","pirate","yand","clown")
+
+	switch(randomSpooky)
```

Comment:
You may want to make it so that this switch is only selected for if a var on the stone is set, or something (that is to say, by default, the stone will always default to roman unless the random var is set).

As is, every single one of these types is *dramatically* inferior to the roman skeletons. The roman skeletons have two weapons, head armor, and, most importantly, a shield.

clown effectively has no weapon (and is slow) and pirate gets a weapon that does 10 damage less than the roman or yand.